### PR TITLE
Add more detail to Readme.md for new users

### PR DESCRIPTION
**Pull request by jknowles**
Minor improvements to the readme to make it easier for users to get started using the program.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,12 +13,42 @@ net inst eda, from(http://wbuchanan.github.io/eda)
 ## Use case
 Since Exploratory Data Analysis can take a substantial amount of time in addition to the time needed to clean/prep data, this is intended to be used as a program that would be called at the end of the workday/overnight to produce permutations of univariate and bivariate visualizations and tables.  Then instead of spending time coding myriad possible combinations of variables to examine, a researcher could browse through a PDF generated through LaTeX while the computer does the work of compiling the results for them.
 
+## Usage
+
+You need a dataset open in memory to use `eda`. There are two required options, `output` for the naming of the output files
```
Comment (wbuchanan) [inline README.md:18]: There is also `using` syntax available as well.  If a user were to issue:

```Stata
eda using auto.dta, o("eda-report") root("./") 
```
It should load the `auto.dta` data set in the current working directory and execute the remainder of the program.

Comment (wbuchanan) [reply README.md:18]: That said, I definitely appreciate the help putting together some more informative documentation.

File: README.md [hunk 2]
```diff
@@ -13,12 +13,42 @@ net inst eda, from(http://wbuchanan.github.io/eda)
...
+
+You can install these dependencies using:
+
+```Stata
+ssc install tuples
+ssc install spineplot
+ssc install estout
+ssc install brewscheme
+```
+
```
Comment (wbuchanan) [inline README.md:51]: Or if these dependencies are not available, `eda` will prompt you to install each of the programs the first time you execute it.

**Review by wbuchanan — CHANGES_REQUESTED** [review]
I added a couple of additional clarifying comments that I thought might also be useful to include.  Not sure if you think it'd be easier to add them into your PR or if you'd prefer adding that stuff later, but I at least wanted to make sure to make a note of things before forgetting them.

File: README.md (hunk 1 above)
Comment (jknowles) [reply README.md:18]: Sorry I didn't get to this in time!

Comment (wbuchanan) [reply README.md:18]: No worries.  I ended up finding another small bug a week or so later that I patched locally but still need to push.

**Pull request merged**